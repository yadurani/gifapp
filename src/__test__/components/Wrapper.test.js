import React from 'react';
import { shallow } from 'enzyme';
import GifExpertAppWrapper from '../../components/GifExpertApp/Wrapper';

describe('Probando el componente <GifExpertAppWrapper/>', () => {
  it('Debe hacer el toMatchSnapshot', () => {
    const wrapper = shallow(<GifExpertAppWrapper/>)
    expect(wrapper).toMatchSnapshot()
  })
  it('Debe de mostrar una lista de categorias', () => {
    const categories = ['One Punch', 'Dragon Ball']
    const wrapper = shallow(<GifExpertAppWrapper defaultCategories={categories}/>)
    expect(wrapper).toMatchSnapshot()
    expect(wrapper.find('GridList').length).toBe(categories.length)
  })
})