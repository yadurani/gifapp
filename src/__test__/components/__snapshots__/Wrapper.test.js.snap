// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probando el componente <GifExpertAppWrapper/> Debe de mostrar una lista de categorias 1`] = `
<div
  className="container"
>
  <h2
    className="title"
  >
    .GIFAPP.
  </h2>
  <AddCategory
    handleAdd={[Function]}
  />
  <ol>
    <GridList
      category="One Punch"
      key="One Punch"
    />
    <GridList
      category="Dragon Ball"
      key="Dragon Ball"
    />
  </ol>
  <small>
    Create by Yadu LÃ³pez ðŸ’›. Github 
    <a
      href="https://github.com/yadurani/gifapp"
      rel="noreferrer"
      target="_blank"
    >
      GIFAPP
    </a>
    .
  </small>
</div>
`;

exports[`Probando el componente <GifExpertAppWrapper/> Debe hacer el toMatchSnapshot 1`] = `
<div
  className="container"
>
  <h2
    className="title"
  >
    .GIFAPP.
  </h2>
  <AddCategory
    handleAdd={[Function]}
  />
  <ol />
  <small>
    Create by Yadu LÃ³pez ðŸ’›. Github 
    <a
      href="https://github.com/yadurani/gifapp"
      rel="noreferrer"
      target="_blank"
    >
      GIFAPP
    </a>
    .
  </small>
</div>
`;
